### PR TITLE
Change the year in the footer

### PR DESCRIPTION
**Pull request by jpbulman**
Makes the year in the footer of pages the current year, rather than having to change it each time.

### CODE REVIEW

**Comment by jpbulman** [comment]
Bumping this

File: src/common.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 from src.models.wca.export import get_latest_export
 
 class Common(object):
+  
+  currentDate = datetime.datetime.now()
```
Comment (timreyn) [inline src/common.py]: nit: the rest of Common uses underscores (current_date), use that for consistency.

File: src/templates/base.html [hunk 2]
```diff
@@ -88,7 +88,7 @@
   <hr>
   <div class="container">
     <footer>
-      © CubingUSA 2017.
+      © CubingUSA {% print(c.currentDate.year) %}.
```
Comment (timreyn) [inline src/templates/base.html]: Jinja lets you use

{{ c.currentDate.year }}

as a shorthand instead of print

**Review by timreyn — CHANGES_REQUESTED** [review]
Sorry for the delay!  Looks good, a few minor points.

File: src/templates/base.html (hunk 2 above)
Comment (jpbulman) [reply src/templates/base.html]: Very cool!

**Comment by jpbulman** [comment]
Made the necessary changes, let me know if this needs anything else!

File: src/templates/base.html [hunk 3]
```diff
@@ -88,7 +88,7 @@
   <hr>
   <div class="container">
     <footer>
-      © CubingUSA 2017.
+      © CubingUSA {{ c.currentDate.year }}.
```
Comment (timreyn) [inline src/templates/base.html]: should now be current_date

Comment (jpbulman) [reply src/templates/base.html]: Ah I knew I should have run it locally to make sure, should be fixed now

**Review by timreyn — APPROVED** [review]

**Pull request merged**